gate-web/config/gate.yml

```diff
@@ -48,6 +49,9 @@ services:
     enabled: false
     canaryConfigStore: true
     baseUrl: http://localhost:8090
+  gremlin:
+    enabled: true
```

Comment:
It doesn't seem correct that Gremlin would default to on

Reply:
I'd suggest a new block:

```
integrations:
  gremlin:
    baseUrl: https://api.gremlin.com/v1
    enabled: true|false
```

Reply:
Makes sense @ajordens and thanks for the review @avram .  Is the following what you have in mind?

- Create a new block `integrations` in `gate.yml`
- Move `GremlinController` and `GremlinService` into a new top-level module `gate-integrations-gremlin`.
- Fix the dependency injection to respect the `integrations.gremlin.enabled` flag and only instantiate `GremlinController` and `GremlinService` if that flag is true.

I'm guessing similar thoughts apply to the Gremlin work in `orca`?  I'm not as familiar with how we can separate out the UI changes in `deck` - any thoughts on how that should get modularized?  In the meantime, will start working through the `gate` + `orca` changes.